src/server/api.ts

```diff
@@ -62,7 +62,6 @@ router.get(["/utdanning", "/:lang/utdanning"], (_, res) => {
 });
 
 router.get("/ukr", (_req, res) => {
-  res.cookie(STORED_LANGUAGE_COOKIE_KEY, "en");
```

Comment:
Skal vi ikke fjerne `/en`-prefixen her også?

Reply:
Nei, for vi må fremdeles støtte visning av engelsk språkdrakt. Meiner lenka i footer for ukrainsk også skal ha /en